src/logic/operate.js

```diff
@@ -1,30 +1,26 @@
 import Big from 'big.js';
 
-const operate = (numberOne, numberTwo, operation) => {
-  const oneBig = Big(numberOne);
-  const twoBig = Big(numberTwo);
-
-  if (operation === '-') {
-    return oneBig.minus(twoBig);
-  }
-
-  if (operation === '+') {
-    return oneBig.plus(twoBig);
-  }
-
-  if (operation === '/' && numberTwo) {
-    return oneBig.div(twoBig);
-  }
-
-  if (operation === '/' && numberTwo === 0) {
-    return Infinity;
-  }
-
-  if (operation === 'X') {
-    return oneBig.mul(twoBig);
+const operate = (numberOne, numberTwo, operator) => {
+  const first = Big(numberOne);
+  const second = Big(numberTwo);
+  let num = '';
+  switch (operator) {
+    case '+': num = first.plus(second).toString();
+      break;
+    case '-': num = first.sub(second).toString();
+      break;
+    case '*': num = first.mul(second).toString();
+      break;
+    case '/':
+      num = numberTwo === '0' ? '0' : first.div(second).toString();
+      break;
+    case '%': num = first.mod(second).toString();
+      break;
```

Comment:
- This should not be mod, it should convert `first` to a percentage based number.  
`95%` should be `0.95`
![Screenshot](https://user-images.githubusercontent.com/16196289/90116728-984bf300-dd5e-11ea-83d7-f9c802e87988.png)